zwave_js_server/model/firmware.py

```diff
@@ -13,6 +13,67 @@
     from .node import Node
 
 
+class FirmwareUpdateCapabilitiesDataType(TypedDict, total=False):
+    """Represent a firmware update capabilities dict type."""
+
+    firmwareUpgradable: bool  # required
+    firmwareTargets: List[int]
+    continuesToFunction: Union[bool, str]
+    supportsActivation: Union[bool, str]
+
+
+class FirmwareUpdateCapabilities:
```

Comment:
We could describe the response with a pydantic model directly.

Reply:
if we are going to do that should we be looking at that elsewhere as well? I almost feel like it should be done as a single big initiative like we did with the event models, what do you think?

Reply:
We should probably do it in the same release, yes. We don't need to do everything in the same PR, I think.

Reply:
Good point